src/metadata.rs

```diff
@@ -604,13 +812,16 @@ mod test {
                     "text_color": "#FFFFFF"
                 }
             ]
-        }))
-        .unwrap();
+        }"##,
+        );
+        let _: CredentialMetadata<CoreProfilesMetadata> =
+            serde_path_to_error::deserialize(jd).unwrap();
     }
 
     #[test]
     fn example_credential_metadata_isomdl() {
-        let _: CredentialMetadata<CoreProfilesMetadata> = serde_json::from_value(json!({
+        let jd = &mut serde_json::Deserializer::from_str(
```

Comment:
I'm curious why some of these tests are using raw strings instead of `json!`?

Reply:
Sorry, it was leftover from manual testing, I was tracking deserialization errors and used `serde_path_to_error::deserialize` and `Deserializer` with a string literal instead of the `json!` macro.